govcd/extension_test.go

```diff
@@ -28,3 +30,82 @@ func (vcd *TestVCD) Test_GetExternalNetwork(check *C) {
 	check.Assert(externalNetwork.Name, Equals, networkName)
 	check.Assert(externalNetwork.Type, Equals, expectedType)
 }
+
+func (vcd *TestVCD) Test_CreateExternalNetwork(check *C) {
+	if vcd.skipAdminTests {
+		check.Skip("Configuration org != 'Sysyem'")
+	}
+	networkName := vcd.config.VCD.ExternalNetwork
+	if networkName == "" {
+		check.Skip("No external network provided")
+	}
+
+	externalNetworkRef, err := GetExternalNetworkByName(vcd.client, networkName)
```

Comment:
Why we delete existing one external network?

Reply:
I got the idea from feedback on my previous pull request: https://github.com/vmware/go-vcloud-director/pull/117#discussion_r240193783

But it doesn't look like that change made it all the way through, so faulty memory on my part.  I'll remove this.

Reply:
I think in that case and in this case - right answer is "exit gracefully with a meaningful error message.". We don't delete what we don't expect to find I would say :)